packages/ckeditor5-emoji/src/ui/emojisearchview.ts

```diff
@@ -29,7 +29,7 @@ export default class EmojiSearchView extends View {
 		this.setTemplate( {
 			tag: 'div',
 			attributes: {
-				class: [ 'ck', 'ck-emoji-input' ]
+				class: [ 'ck', 'ck-emoji-input', 'ck-search' ]
```

Comment:
In order for the styles to look correct, the `ck-search` class is needed. So I'm not sure if exposing the search input is a good idea. The component should be self-sustaining, without needing any additional modifications like adding classes, to work.